arch/ARM/Nordic/drivers/nrf51/nrf-twi-master_transmit.adb

```diff
@@ -0,0 +1,65 @@
+separate (nRF.TWI)
```

Comment:
It bothers me to have those `separate` for timer, TWI, etc. just because the SVD uses different names.

Reply:
I agree.

Currently the svd.mk generates from copies of the vendor SVDs stored *in the svd2ada repository*. I had thought about starting a pull request against svd2ada to make these names line up...

I don't know who to contact to "upstream" these sorts of changes at Nordic...

Reply:
So far we already allowed ourselves to change the SVDs in `svd2ada`, so I would recommend to do that as well for the `nRF5x`.


> I don't know who to contact to "upstream" these sorts of changes at Nordic...

I don't know either. The Rust community contributed upstream changes to some vendors but I don't know which ones.